src/ngx_http_lua_socket_tcp.c

```diff
@@ -27,6 +27,11 @@ static int ngx_http_lua_socket_tcp_sslhandshake(lua_State *L);
 static int ngx_http_lua_socket_tcp_receive(lua_State *L);
 static int ngx_http_lua_socket_tcp_receiveany(lua_State *L);
 static int ngx_http_lua_socket_tcp_send(lua_State *L);
+
+// send the exising buffer out non-blocking way.
+// buffer should be set using ngx_http_lua_socket_tcp_ffi_set_buf
```

Comment:
Do not use C++-style comments in this module. Please see the C coding style guide for more details:

https://openresty.org/en/c-coding-style-guide.html